source/angular-js/build.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/env bash
+### PLAYGROUND INTRO START
+set -e # exit when any command fails
+cd "$(dirname "${BASH_SOURCE[0]}")" # move to this script’s directory
+THIS_DIR=$(pwd)
+PROJECT_NAME="$(basename "$THIS_DIR")"
+PROJECT_DIST="$(cd ../../public && pwd)/$PROJECT_NAME"
+echo "Building '$PROJECT_NAME' into '$PROJECT_DIST'"
+### PLAYGROUND INTRO END
+
+cd repo
+npm install
+gulp build
```

Comment:
This only works if gulp is installed locally. It's probably either one of this
```suggestion
npx gulp build
```
```suggestion
npm run build
```